README.md

```diff
@@ -884,4 +884,4 @@ Currently mobile is not supported. However, there are plans to add touch support
 
 ## Author / maintainer
 
-Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
\ No newline at end of file
+Alex Reardon - [@alexandereardon](https://twitter.com/alexandereardon) - areardon@atlassian.com
```

Comment:
I don't know either - I've never seen this with any of the other pull requests I submit. I'll close this and you can do it directly. Sorry for the hassle.